ebpfapi/Source.def

```diff
@@ -96,14 +96,14 @@ EXPORTS
     bpf_xdp_attach
     bpf_xdp_detach
     bpf_xdp_query_id
-    ebpf_api_elf_disassemble_section
-    ebpf_api_elf_verify_section_from_file
-    ebpf_api_elf_verify_section_from_memory
+    ebpf_api_elf_disassemble_program
+    ebpf_api_elf_verify_program_from_file
```

Comment:
Ideally we should ensure all the public APIs are backward compatible. We can probably add new APIs and also keep the older APIs, but mark them as deprecated (and only remove them when there are no more users of the deprecated APIs)

Reply:
Done